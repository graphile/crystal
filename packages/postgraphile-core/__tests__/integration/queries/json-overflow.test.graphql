# Tests passing more than 100 values to json_build_object
query {
  allPosts(first: 1) {
    nodes {
      headline
      a1: headlineTrimmed(length: 1)
      a2: headlineTrimmed(length: 2)
      a3: headlineTrimmed(length: 3)
      a4: headlineTrimmed(length: 4)
      a5: headlineTrimmed(length: 5)
      a6: headlineTrimmed(length: 6)
      a7: headlineTrimmed(length: 7)
      a8: headlineTrimmed(length: 8)
      a9: headlineTrimmed(length: 9)
      a10: headlineTrimmed(length: 10)
      a11: headlineTrimmed(length: 11)
      a12: headlineTrimmed(length: 12)
      a13: headlineTrimmed(length: 13)
      a14: headlineTrimmed(length: 14)
      a15: headlineTrimmed(length: 15)
      a16: headlineTrimmed(length: 16)
      a17: headlineTrimmed(length: 17)
      a18: headlineTrimmed(length: 18)
      a19: headlineTrimmed(length: 19)
      a20: headlineTrimmed(length: 20)
      a21: headlineTrimmed(length: 21)
      a22: headlineTrimmed(length: 22)
      a23: headlineTrimmed(length: 23)
      a24: headlineTrimmed(length: 24)
      a25: headlineTrimmed(length: 25)
      a26: headlineTrimmed(length: 26)
      a27: headlineTrimmed(length: 27)
      a28: headlineTrimmed(length: 28)
      a29: headlineTrimmed(length: 29)
      a30: headlineTrimmed(length: 30)
      a31: headlineTrimmed(length: 31)
      a32: headlineTrimmed(length: 32)
      a33: headlineTrimmed(length: 33)
      a34: headlineTrimmed(length: 34)
      a35: headlineTrimmed(length: 35)
      a36: headlineTrimmed(length: 36)
      a37: headlineTrimmed(length: 37)
      a38: headlineTrimmed(length: 38)
      a39: headlineTrimmed(length: 39)
      a40: headlineTrimmed(length: 40)
      a41: headlineTrimmed(length: 41)
      a42: headlineTrimmed(length: 42)
      a43: headlineTrimmed(length: 43)
      a44: headlineTrimmed(length: 44)
      a45: headlineTrimmed(length: 45)
      a46: headlineTrimmed(length: 46)
      a47: headlineTrimmed(length: 47)
      a48: headlineTrimmed(length: 48)
      a49: headlineTrimmed(length: 49)
      a50: headlineTrimmed(length: 50)
      a51: headlineTrimmed(length: 51)
      a52: headlineTrimmed(length: 52)
      a53: headlineTrimmed(length: 53)
      a54: headlineTrimmed(length: 54)
      a55: headlineTrimmed(length: 55)
      a56: headlineTrimmed(length: 56)
      a57: headlineTrimmed(length: 57)
      a58: headlineTrimmed(length: 58)
      a59: headlineTrimmed(length: 59)
      a60: headlineTrimmed(length: 60)
      a61: headlineTrimmed(length: 61)
      a62: headlineTrimmed(length: 62)
      a63: headlineTrimmed(length: 63)
      a64: headlineTrimmed(length: 64)
      a65: headlineTrimmed(length: 65)
      a66: headlineTrimmed(length: 66)
      a67: headlineTrimmed(length: 67)
      a68: headlineTrimmed(length: 68)
      a69: headlineTrimmed(length: 69)
      a70: headlineTrimmed(length: 70)
      a71: headlineTrimmed(length: 71)
      a72: headlineTrimmed(length: 72)
      a73: headlineTrimmed(length: 73)
      a74: headlineTrimmed(length: 74)
      a75: headlineTrimmed(length: 75)
      a76: headlineTrimmed(length: 76)
      a77: headlineTrimmed(length: 77)
      a78: headlineTrimmed(length: 78)
      a79: headlineTrimmed(length: 79)
      a80: headlineTrimmed(length: 80)
      a81: headlineTrimmed(length: 81)
      a82: headlineTrimmed(length: 82)
      a83: headlineTrimmed(length: 83)
      a84: headlineTrimmed(length: 84)
      a85: headlineTrimmed(length: 85)
      a86: headlineTrimmed(length: 86)
      a87: headlineTrimmed(length: 87)
      a88: headlineTrimmed(length: 88)
      a89: headlineTrimmed(length: 89)
      a90: headlineTrimmed(length: 90)
      a91: headlineTrimmed(length: 91)
      a92: headlineTrimmed(length: 92)
      a93: headlineTrimmed(length: 93)
      a94: headlineTrimmed(length: 94)
      a95: headlineTrimmed(length: 95)
      a96: headlineTrimmed(length: 96)
      a97: headlineTrimmed(length: 97)
      a98: headlineTrimmed(length: 98)
      a99: headlineTrimmed(length: 99)
      a100: headlineTrimmed(length: 100)
      a101: headlineTrimmed(length: 101)
      a102: headlineTrimmed(length: 102)
      a103: headlineTrimmed(length: 103)
    }
  }
}
