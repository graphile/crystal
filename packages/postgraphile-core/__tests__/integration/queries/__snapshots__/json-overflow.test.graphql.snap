// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "allPosts": Object {
      "nodes": Array [
        Object {
          "a1": "…",
          "a10": "No… It’s …",
          "a100": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a101": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a102": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a103": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a11": "No… It’s a…",
          "a12": "No… It’s a …",
          "a13": "No… It’s a t…",
          "a14": "No… It’s a th…",
          "a15": "No… It’s a thi…",
          "a16": "No… It’s a thin…",
          "a17": "No… It’s a thing…",
          "a18": "No… It’s a thing;…",
          "a19": "No… It’s a thing; …",
          "a2": "N…",
          "a20": "No… It’s a thing; i…",
          "a21": "No… It’s a thing; it…",
          "a22": "No… It’s a thing; it’…",
          "a23": "No… It’s a thing; it’s…",
          "a24": "No… It’s a thing; it’s …",
          "a25": "No… It’s a thing; it’s l…",
          "a26": "No… It’s a thing; it’s li…",
          "a27": "No… It’s a thing; it’s lik…",
          "a28": "No… It’s a thing; it’s like…",
          "a29": "No… It’s a thing; it’s like …",
          "a3": "No…",
          "a30": "No… It’s a thing; it’s like a…",
          "a31": "No… It’s a thing; it’s like a …",
          "a32": "No… It’s a thing; it’s like a p…",
          "a33": "No… It’s a thing; it’s like a pl…",
          "a34": "No… It’s a thing; it’s like a pla…",
          "a35": "No… It’s a thing; it’s like a plan…",
          "a36": "No… It’s a thing; it’s like a plan,…",
          "a37": "No… It’s a thing; it’s like a plan, …",
          "a38": "No… It’s a thing; it’s like a plan, b…",
          "a39": "No… It’s a thing; it’s like a plan, bu…",
          "a4": "No……",
          "a40": "No… It’s a thing; it’s like a plan, but…",
          "a41": "No… It’s a thing; it’s like a plan, but …",
          "a42": "No… It’s a thing; it’s like a plan, but w…",
          "a43": "No… It’s a thing; it’s like a plan, but wi…",
          "a44": "No… It’s a thing; it’s like a plan, but wit…",
          "a45": "No… It’s a thing; it’s like a plan, but with…",
          "a46": "No… It’s a thing; it’s like a plan, but with …",
          "a47": "No… It’s a thing; it’s like a plan, but with m…",
          "a48": "No… It’s a thing; it’s like a plan, but with mo…",
          "a49": "No… It’s a thing; it’s like a plan, but with mor…",
          "a5": "No… …",
          "a50": "No… It’s a thing; it’s like a plan, but with more…",
          "a51": "No… It’s a thing; it’s like a plan, but with more …",
          "a52": "No… It’s a thing; it’s like a plan, but with more g…",
          "a53": "No… It’s a thing; it’s like a plan, but with more gr…",
          "a54": "No… It’s a thing; it’s like a plan, but with more gre…",
          "a55": "No… It’s a thing; it’s like a plan, but with more grea…",
          "a56": "No… It’s a thing; it’s like a plan, but with more great…",
          "a57": "No… It’s a thing; it’s like a plan, but with more greatn…",
          "a58": "No… It’s a thing; it’s like a plan, but with more greatne…",
          "a59": "No… It’s a thing; it’s like a plan, but with more greatnes…",
          "a6": "No… I…",
          "a60": "No… It’s a thing; it’s like a plan, but with more greatness…",
          "a61": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a62": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a63": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a64": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a65": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a66": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a67": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a68": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a69": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a7": "No… It…",
          "a70": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a71": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a72": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a73": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a74": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a75": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a76": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a77": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a78": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a79": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a8": "No… It’…",
          "a80": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a81": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a82": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a83": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a84": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a85": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a86": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a87": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a88": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a89": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a9": "No… It’s…",
          "a90": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a91": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a92": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a93": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a94": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a95": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a96": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a97": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a98": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "a99": "No… It’s a thing; it’s like a plan, but with more greatness.…",
          "headline": "No… It’s a thing; it’s like a plan, but with more greatness.",
        },
      ],
    },
  },
}
`;
